modules/openapi-generator/src/main/resources/python/model_templates/method_from_openapi_data_composed.mustache

```diff
@@ -0,0 +1,66 @@
+    @classmethod
+    @convert_js_args_to_python_args
+    def from_openapi_data(cls, *args, **kwargs):  # noqa: E501
```

Comment:
What if someone wanted to name a property from_openapi_data?
How about using `_from_openapi_data` to reduce the likelihood of a naming collision?

Reply:
ok

Reply:
I guess now I change the target branch to master since 5.2 was merged today